frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -319,7 +319,7 @@ export class NativeQueryEditor extends Component<
 
   componentWillUnmount() {
     if (this.props.cancelQueryOnLeave) {
-      this.props.cancelQuery?.();
+      // this.props.cancelQuery?.();
```

Comment:
FIXME. The issue is that when we start doing native query drills, this line cancels the new query. Find another solution. Is this needed? Maybe the QB already cancels the query correctly?

Reply:
we'll see from tests 😄